.github/workflows/python-test-application.yml

```
@@ -0,0 +1,35 @@
+name: Python Tests for Applications
+
+on:
+  workflow_call:
+    inputs:
+      PYTHON_VERSION:
+        required: true
+        type: string
+        description: The python version to test and build against
```

```suggestion
        description: The python version to test against
```
We're not building apps, only testing